grub/plan.sh

```diff
@@ -6,7 +6,7 @@ pkg_shasum=TODO
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_description="GNU GRUB is a Multiboot boot loader."
 pkg_upstream_url=https://www.gnu.org/software/grub/
-pkg_license=('Apache-2.0')
+pkg_license=('GPLv3')
```

Comment:
It should be "GPL-3.0" according to SPDX: https://spdx.org/licenses/GPL-3.0.html